keps/sig-cli/0014-krew.md

```diff
@@ -0,0 +1,237 @@
+---
+kep-number: 14
+title: Krew
```

Comment:
The name confused me when I first looked at this. I understand that krew is the name of the proof of concept. Should the title be more expressive of the purpose?